src/test/java/de/tum/cit/aet/artemis/iris/IrisChatMessageIntegrationTest.java

```diff
@@ -448,101 +455,6 @@ void sendMessageRateLimitReached() throws Exception {
         }
     }
 
-    private IrisMessage createDefaultMockMessage(IrisSession irisSession) {
-        var messageToSend = irisSession.newMessage();
-        messageToSend.addContent(createMockTextContent(), createMockTextContent(), createMockTextContent());
-        return messageToSend;
```

Comment:
This is duplicated code, you will see it in the following files aswell - I moved this functionality to the util services and factories